### PR TITLE
Re-former project following the Odin Project Instructions

### PR DESCRIPTION
**Pull request by jadx2**
We created a model for the users with its corresponding controller to create and edit.
Also, the views to interact with the database from the browser.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
+  resources :users
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
```
Comment (blackpintz) [inline config/routes.rb:3]: Use the `:only` option to specify only the routes used in this application. See screenshot [one](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_route1.png) and [two](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_route2.png) of this requirement.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```
Comment (blackpintz) [inline app/controllers/users_controller.rb:7]: Remove the commented code.

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+<%
+=end%>
+
+<%
+=begin%>
+ <%= form_tag("/users", method: "post") do %>
+  <%= label_tag(:username, "Name") %>
+  <%= text_field_tag(:username) %>
+  <%= label_tag(:email, "Email") %>
+  <%= email_field(:user, :email) %>
```
Comment (blackpintz) [inline app/views/users/new.html.erb:22]: Write `email_field_tag` properly.

**Review by blackpintz — CHANGES_REQUESTED** [review]
# Status: Changes Required 🛑

### Hello team 👋,

**Great job on your project 👏.**

### Highlights

- **Your README is professional.**
- **You have passed most of the requirements.**

### Mandatory Requirements

- **Please check the comments below.**

### Happy coding 🥂.

**You can reach me on slack (Rose Wanjohi).**

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (elmejdki) [reply app/controllers/users_controller.rb:7]: remove the commented code in the users_controller

File: app/views/users/new.html.erb (hunk 3 above)
Comment (elmejdki) [reply app/views/users/new.html.erb]: add _tag to the email_field in the new.html.erb

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 

I think the requirements were really clear by the previous TSE, please revert your changes to the new.html.erb file and work on the comments that I have added to the previous review.
:point_up: :point_up: :point_up: :point_up: :point_up:

### Please make all the required changes, before you submit another code review.
## Happy Coding :tophat:

**Review by LamiaSristy — APPROVED** [review]
# Status: Approved. 🎉 🎉

Hello @jadx2 and @RokoVarano. 

**To get professionalism sometimes people need to work hard. Thank you for making the changes the former reviewers asked for.**

**After all the changes and your hard work now you have a really great project, congratulations on that.**

You can merge the branch now.

good luck with your next Project 🚀.

Feel free to reach me on slack @lamiasristy 🙋

Keep up the spirit always. 

Happy Coding 💻

**Pull request merged**